doc/api/esm.md

```diff
@@ -1130,18 +1130,42 @@ node --experimental-top-level-await b.mjs # works
 <!-- type=misc -->
 
 To customize the default module resolution, loader hooks can optionally be
-provided via a `--experimental-loader ./loader-name.mjs` argument to Node.js.
-
+provided via `--experimental-loader ./loader-name.mjs` arguments to Node.js.
 When hooks are used they only apply to ES module loading and not to any
 CommonJS modules loaded.
 
+Loaders are modules which export functions and data which can be used by
+Node.js to modify the behavior of loading ES Modules. Loaders may be ”chained”
+together by passing multiple `--experimental-loader` arguments to Node.js. A
+loader module will be imported once as a singleton.
+
+A hook has access to an additional `nextLoader` argument, which contains
+functions which call the hooks of the next loader. When calling the next hook,
+`nextLoader` **does not** need to be passed.
+
+### <code>nextLoader</code>
+
+* {Object}
+  * `resolve` {Function}
+  * `getFormat` {Function}
+  * `getSource` {Function}
```

Comment:
There are hooks missing from this object.

Reply:
the "next"/"previous"/"parent" concept doesn't apply to `getGlobalPreloadCode` or to `transformSource`, so they aren't included.

Reply:
Unclear on why that would be the case. Let's say I have a loader that needs access to the string returned by the `getGlobalPreloadCode` hook of the subsequent loader in order for it to work. There doesn't seem to be a reason why such behavior shouldn't be possible seeing as how similar access is allowed to the other hooks.

Reply:
each loader can pass its own global preload code, it's not something that "chains"

Reply:
Is the string returned by the `getGlobalPreloadCode` hook of the subsequent loader intended to be a secret?

Reply:
@DerekNonGeneric i wouldn't say "secret" but i can't imagine any good pattern that would come out of exposing it to other loaders